### DISCUSSION TITLE
What should I do to get work M300?

### DISCUSSION THREAD

**Discussion by sg1oneill** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2396] [Q&A]
I know it is almost 2 years since M300 support was added, but I have just installed my first TFT - it is very good by the way and everything worked well, except M300.   The Buzzer works, if I touch the screen but M300 does not. Doesn't matter if I use the screen gcode section and type it manually or pronterface. It just does nothing.
Could someone give me some direction on how to fix it?
I am using the latest BIGTREE_TFT35_V3.0_E3.27.x.bin relase. 
Thank you.

**Comment by kisslorand** [comment]
I just tested it, in the Terminal I typed M300 P500 S5000 and the TFT gave me a tone.
To what is Pronterface connected to? If it is connected to the motherboard than you send M300 command to the motherboard, not the TFT.

**Reply by sg1oneill** [reply]
Thank you for your help. I am not sure why it did not work, but now it does. I am doing lots of firmware uploads as something is always wrong :D So who knows, maybe I was uploading the wrong one, I am really not sure.  Anyway, it is OK now. 
By the way, I am printing and tunning my printers for a while, Changed and used screens before many times, but not a Bigtreetech one. I know Pronterface and I understand the different between the TFT and the motherboard :) 
Thank you again!
